rectools/models/base.py

```diff
@@ -194,6 +194,10 @@ def recommend(
         reco_warm_final = self._reco_to_external(reco_warm, dataset.user_id_map, dataset.item_id_map)
         reco_cold_final = self._reco_items_to_external(reco_cold, dataset.item_id_map)
 
+        reco_hot_final = self._adjust_reco_types(reco_hot_final, dataset.user_id_map.external_dtype, item_type)
```

Comment:
Let's just pass it like `dataset.item_id_map.external_dtype` just like we are doing for users

Reply:
dataset.item_id_map has type object after _custom_transform_dataset_u2i